seds/mongodb-sed/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "mysqlsecret.name" -}}
```

Comment:
change mysqlsecret to mongodb-sed across this document

Reply:
changed it